sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -9,6 +9,76 @@ import { AbortSignalLike } from '@azure/abort-controller';
 // @public
 export type CancelOnProgress = () => void;
 
+// @public
+export function createGetLROStatusFromResponse<TResult>(lroPrimitives: LRO<TResult>, config: LROConfig, finalStateVia?: FinalStateVia): (rawResponse: RawResponse, flatResponse: TResult) => LROStatus<TResult>;
```

Comment:
Should we reuse the below type alias for the return type of this function? 

e.g.

```suggestion
export function createGetLROStatusFromResponse<TResult>(lroPrimitives: LRO<TResult>, config: LROConfig, finalStateVia?: FinalStateVia): GetLROStatusFromResponse<TResult>;
```

Reply:
Done.